samples/SkiaMultiplatformSample/src/macosMain/kotlin/org/jetbrains/skiko/sample/App.macos.kt

```diff
@@ -1,13 +1,27 @@
 package org.jetbrains.skiko.sample
 
+import kotlinx.coroutines.*
 import platform.AppKit.*
 
 import org.jetbrains.skiko.*
-import platform.Foundation.NSMakeRect
-import platform.Foundation.NSSelectorFromString
-import platform.darwin.NSObject
+import org.jetbrains.skiko.notifications.Notification
+import platform.Foundation.*
+import platform.darwin.*
 
-fun makeApp(skiaLayer: SkiaLayer) = Clocks(skiaLayer)
+fun makeApp(skiaLayer: SkiaLayer) = object : Clocks(skiaLayer) {
+    override fun onKeyboardEvent(event: SkikoKeyboardEvent) {
+        super.onKeyboardEvent(event)
+        if (event.kind == SkikoKeyboardEventKind.DOWN) when (event.key) {
+            SkikoKey.KEY_N -> runBlocking {
+                Notification(
```

Comment:
Hmm, why `runBlocking`? Cannot we just run it as suspend on `SkikoDispatchers.Main` dispatcher?